### DISCUSSION TITLE
The order of a common localSpan and another localSpan for hystrix is not right

### DISCUSSION THREAD

**Discussion by lingli0219** [apache/skywalking#8949] [General]
![微信图片_10](https://user-images.githubusercontent.com/31304902/165212842-98ab5452-734e-4b27-9972-b2f67960b3bf.jpg)
Just like the image above shows, the order of a common localSpan and the localSpan for Hystrix is not consistent with the timestamp of both. The image is from the default url localhost:8080/trace of Skywalking. I can't tell why. Please help me.

**Comment by Fine0830** [comment]
Could you check the response data and see which are the parent elements of these two spans?

**Reply by lingli0219** [reply]
![微信图片_11](https://user-images.githubusercontent.com/31304902/165227334-e3915c99-ba46-435e-862b-c6be02ddcff9.jpg)
This is the parent element of the two spans. Thank you for your reply.

**Reply by Fine0830** [reply]
I can't reproduce the problem. Could you provide the http response data here?

**Reply by lingli0219** [reply]
![微信图片_20](https://user-images.githubusercontent.com/31304902/165236393-4972256a-7986-4d00-9ed6-7965a1559d81.jpg)
![微信图片_21](https://user-images.githubusercontent.com/31304902/165236416-678483cf-6aee-423d-a713-ff3165a31f74.jpg)
![微信图片_22](https://user-images.githubusercontent.com/31304902/165236433-ddf252d3-b81e-4296-9040-701758269ae9.jpg)
![微信图片_23](https://user-images.githubusercontent.com/31304902/165236489-81bf6ba4-1682-497b-8113-67f7b88686cf.jpg)
![微信图片_24_2](https://user-images.githubusercontent.com/31304902/165237256-36edfaa3-47cb-48ee-bf7c-afa792fa8722.jpg)
![微信图片_25](https://user-images.githubusercontent.com/31304902/165237267-557cd780-4376-4760-96da-d3c0c9386e69.jpg)
The first 2 images are in the list form and the latter 4 images are in the table form. You can see in the table form (the first four lines of the fourth image) that the order is not consistent with time.

**Reply by wu-sheng** [reply]
response from your browser logs. Otherwise, it is impossible to reproduce.

**Reply by wu-sheng** [reply]
Screenshots are just demonstrating. If you want us to address this, provide data.

**Reply by lingli0219** [reply]
I don't know what are browser logs. Can you tell me how to find them?

**Reply by Fine0830** [reply]
Open the Network on your browser.

<img width="850" alt="1" src="https://user-images.githubusercontent.com/20871783/165263239-93f48fdc-c43d-41fa-9f32-9e126c471e13.png">

**Reply by lingli0219** [reply]
![微信图片_30](https://user-images.githubusercontent.com/31304902/165288397-ae08044a-516e-4cf2-8aa4-e96f81be8c88.jpg)
![微信图片_31](https://user-images.githubusercontent.com/31304902/165288414-722166cd-cfc1-4cf2-8bd9-392b05d4c27d.jpg)
![微信图片_32](https://user-images.githubusercontent.com/31304902/165288425-af55da1f-58e2-453c-ba70-8ce3876ecf8a.jpg)
Sorry I can't access the network through my computer for working. There is a lot of data from the response data and I can't take photos of all of the data. The above is almost all I can give. The first image shows where goes wrong. And the latter two images show the context data of where went wrong. Please let me know if you need more information.

**Reply by wu-sheng** [reply]
You should know, community does not provide services, we have asked minimam to locate possible bugs, if you can't, there is nothing should be done through our side.
